examples/hyrax.yaml

```diff
@@ -7,597 +7,429 @@
-#  usage_guidelines: 
-  requirement: recommended
-#  controlled_values:
-#    controlled_value_format: 
-#    controlled_value_sources: 
-  sample_value: "This item available for access on campus only."
-  property_uri:  http://purl.org/dc/terms/accessRights #https://github.com/samvera/hyrax/issues/3295
-  available_on_object:
-    - GenericWork
-    - Image
-  validations:
-    range: http://www.w3.org/2000/01/rdf-schema#Literal
-    data_type: http://www.w3.org/2001/XMLSchema#string
-#    syntax:
-#    validation_regex:
-    minimum_occurrence: 0
-#    maximum_occurrence: 
-#  index_documentation: 
-#  mapping:
-#    service: 
-#    property_name: 
+  - resource_type:
+      display_label: "Resource type"
+      definition: "Pre-defined categories to describe the type of content being uploaded, such as "article" or "dataset." More than one type may be selected."
+      requirement: recommended
+      controlled_values:
+        controlled_value_format: http://www.w3.org/2000/01/rdf-schema#Literal
+        controlled_value_sources: "local/path/to/yaml"
+      sample_value: "Article"
+      property_uri: http://purl.org/dc/terms/type
+      available_on:
+        class:
+          - GenericWork
+          - Image
+      range: http://www.w3.org/2000/01/rdf-schema#Literal
+      data_type: http://www.w3.org/2001/XMLSchema#string
+      cardinality:
+        minimum: 0
+        maximum:
+      index_documentation: "Resource type should be indexed as searchable, displayable, and facetable."
+      mapping:
+        dpla: "http://purl.org/dc/terms/type"
 
-- property_method_name: abstract
-  display_label: "Abstract"
-  definition: "A summary of the resource. Examples include abstracts of a paper or citation information for a journal article."
-#  usage_guidelines: 
-  requirement: recommended
-#  controlled_values:
-#    controlled_value_format: 
-#    controlled_value_sources: 
-  sample_value: "This is my article abstract. It tells you what the article is going to tell you. This is not a real abstract, it's an abstraction."
-  property_uri:  http://purl.org/dc/terms/abstract #https://github.com/samvera/hyrax/issues/3297 (this is Done in Hyrax 3.0.0)
-  available_on_object:
-    - GenericWork
-    - Image
-  validations:
-    range: http://www.w3.org/2000/01/rdf-schema#Literal
-    data_type: http://www.w3.org/2001/XMLSchema#string
-#    syntax:
-#    validation_regex:
-    minimum_occurrence: 0
-#    maximum_occurrence: 
-#  index_documentation: 
-#  mapping:
-#    service: 
-#    property_name: 
+  - title:
+      display_label: "Title"
+      definition: "For GenericWork and Image: A name to aid in identifying a work or collection. For Collection: A name to aid in identifying the Administrative Set and to distinguish it from other Administrative Sets in the repository."
+    #  definition: "A name to aid in identifying the Administrative Set and to distinguish it from other Administrative Sets in the repository." #Collection
```

Comment:
I think this is the correct way to do this right now (I vaguely remember discussing this at some point in the past). This keeps the spec from being overly complex.